packages/manager/apps/dedicated/client/app/components/dedicated-cloud/users/vsphere-users/delete/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,6 @@
+{
+  "dedicatedCloud_USER_delete_title": "Supprimer un utilisateur",
+  "dedicatedCloud_USER_delete_message": "Etes-vous certain(e) de vouloir supprimer l'utilisateur <b>{{t0}}</b>?",
```

Comment:
**nitpick:** remove html from TRANS files

Reply:
It's applied.